tests/dtm/test_single_process_restart.py

```diff
@@ -631,3 +641,204 @@ def test_mpu_m0d_restart(self):
                       "m0d recovery")
 
         self.log.info("ENDED: Verify multipart upload and download before/after m0d is restarted")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-41229")
+    def test_write_read_delete_during_m0d_restart(self):
+        """Verify WRITE, READ, DELETE during m0d restart using pkill."""
+        self.log.info("STARTED: Verify WRITE, READ, DELETE during m0d restart using pkill")
+        bucket_name = 'bucket-test-41219'
+        object_prefix = 'object-test-41219'
+        log_file_prefix = 'test-41219'
+
+        que = multiprocessing.Queue()
+
+        self.log.info("Step 1: Start Write Operations for parallel reads "
+                      "and parallel deletes during m0d restart:")
+        write_proc = []
+        for _ in range(0, 2):
+            proc = multiprocessing.Process(target=self.dtm_obj.perform_write_op,
+                                           args=(bucket_name, object_prefix,
+                                                 self.test_cfg['clients'],
+                                                 self.test_cfg['samples'],
+                                                 self.test_cfg['size'],
+                                                 log_file_prefix,
+                                                 que))
+            proc.start()
+            write_proc.append(proc)
+
+        self.log.info("Step 2: Wait for Write Operation to complete.")
+        workload_info_list = []
+        for each in write_proc:
+            if each.is_alive():
+                each.join()
+            resp = que.get()
+            assert_utils.assert_true(resp[0], resp[1])
+            workload_info_list.append(resp[1])
+
+        self.log.info("Step 3: Perform Write,Read,Delete Parallely during m0d restart")
```

Comment:
all this happening on one bucket only?

Reply:
No, it will be done on different buckets.